src/main/scala/org/openeo/opensearch/OpenSearchResponses.scala

```diff
@@ -203,6 +204,14 @@ object OpenSearchResponses {
       gdalPrefix
     }
 
+    private def normalizePath(fileLocation: Object): String = {
+      var str = Paths.get(fileLocation.toString).normalize().toString
+      if (System.getProperty("os.name").toLowerCase.contains("windows")) {
+        str = str.replace("\\", "/")
```

Comment:
So we now convert the windows-style path back into a unix path right? The tricky part is perhaps: what if the backend would actually run on windows?

Reply:
Would it be OK to state in the README Windows is not supported?
This change makes the auto tests work on Windows, but I could not test it in an integration.

Reply:
Maybe the problem is that the original code was treating this uri as a posix path:
 Paths.get(fileLocation.toString).normalize()
Perhaps we can find an alternative for this 'normalize' that treats the string more like a url?

Reply:
Maybe this:
URI.create(fileLocation.toString).normalize()
Only problem is that I'm not exactly sure why this normalize was added, and if it's properly covered by tests, but replacing with URI normalize seems like the better option here...

Reply:
fixed. If something Linux related would crash here for, we would find out soon by auto tests or the tests you would be running ^^